src/org/labkey/test/components/ui/permissions/GroupRow.java

```diff
@@ -0,0 +1,99 @@
+package org.labkey.test.components.ui.permissions;
+
+import org.labkey.test.Locator;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class GroupRow extends PermissionsRowBase<GroupRow>
+{
+    protected GroupRow(WebElement el, WebDriver driver)
+    {
+        super(el, driver);
+    }
+
+    @Override
+    protected WebElement getAssignmentContainer()
+    {
+        return Locator.tagWithClass("div", "expandable-container__member-buttons")
+                .waitForElement(getComponentElement(), 2000);
+    }
+
+    @Override
+    protected GroupRow getThis()
+    {
+        return this;
+    }
+
+    protected boolean isDeleteEmptyButtonDisabled()
+    {
+        return elementCache().deleteEmptyGroupBtn().getAttribute("class")
+                .contains("disabled-button-with-tooltip");
```

Comment:
Should be able to use the standard `isEnabled` method.
```suggestion
        return !elementCache().deleteEmptyGroupBtn().isEnabled();
```